src/Sulu/Component/DocumentManager/TimingDocumentManager.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager;
+
+use Metadata\Driver\AdvancedFileLocatorInterface;
+use PHPCR\Query\QueryInterface;
+use Sulu\Component\DocumentManager\Query\Query;
+use Symfony\Component\Stopwatch\Stopwatch;
+use Symfony\Contracts\Service\ResetInterface;
+
+class TimingDocumentManager implements DocumentManagerInterface, ResetInterface
```

Comment:
I just stumbled over https://github.com/sulu/sulu/blob/2.5/src/Sulu/Component/DocumentManager/EventDispatcher/DebugEventDispatcher.php which was build todo such things. As reported in the Slack Channel it is currently broken, so maybe we should just fix the `DebugEventDispatcher` and so avoid the BC Break in the fixtures.

Reply:
That sounds like a good idea. It might still be worth using the interface in the fixtures though but that could also be part of 3.0